versions/1/data_stream/manifest.spec.yml

```diff
@@ -83,6 +86,14 @@ spec:
             default: true
             examples:
               - false
+          url_allowed_schemes:
+            description: Allowed URL schemes for the url type. If not set, any scheme is allowed.
```

Comment:
I think we should call this `Allowed URL protocols` and `url_allowed_protocols` to match the URL spec's terminology

Reply:
The truth is that I took the name from the Go implementation ([url.URL](https://pkg.go.dev/net/url#URL)), but checking the specs, they actually refer to this part as "scheme" (see [here](https://en.wikipedia.org/wiki/URL#Syntax), [here](https://url.spec.whatwg.org/#url-representation), or [here](https://www.w3.org/Addressing/URL/url-spec.txt)).

In any case, as schemes use to refer to protocols, I am happy to change this if it is more clear this way.

Reply:
I stand corrected, thanks!